app/src/main/java/org/literacyapp/receiver/BootReceiver.java

```diff
@@ -26,26 +27,44 @@ public void onReceive(Context context, Intent intent) {
 
         // Initiate background job for synchronizing with server content
         ComponentName componentName = new ComponentName(context, ContentSynchronizationJobService.class);
-        JobInfo.Builder builder = new JobInfo.Builder(1, componentName);
+        JobInfo.Builder builder = new JobInfo.Builder(LiteracyApplication.CONTENT_SYNCRHONIZATION_JOB_ID, componentName);
         builder.setPeriodic(1000 * 60 * 30); // Every 30 minutes
         JobInfo jobInfo = builder.build();
         JobScheduler jobScheduler = (JobScheduler) context.getSystemService(Context.JOB_SCHEDULER_SERVICE);
         jobScheduler.schedule(jobInfo);
 
         // Initiate background job for face recognition training
+        scheduleFaceRecognitionTranining(context);
+
+        // Initiate authentication job for face recognition authentication
+        scheduleAuthentication(context, 30);
+
+        // Start service for detecting when the screen is turned on
+        Intent screenOnServiceIntent = new Intent(context, ScreenOnService.class);
```

Comment:
I think so. You can delete it (better if you do it, because you know the dependencies)